### DISCUSSION TITLE
Development notice - Future of Box, Box!

### DISCUSSION THREAD

**Discussion by BrightDV** [BrightDV/BoxBox#168] [Announcements]
Hello all,

Firstly, thanks to everyone who contributed to this project, suggested a feature, reported a bug… This project was and is something very important to me because I do like working on it. However (you saw it coming), I am entering a new step of my life. Thus, I will not have that much time to add a lot of features in the next couple of years (2 or 3).
I am **not planning to stop the development**, only that I will not be able to add new features, and will be slow to fix bugs. The app has done a full season and more, so it should be ready to work on its own (I hope).

TL;DR: development pace will slow down a lot, but not stop.

PS: from now on, I recommend using the official API, because it is more stable and Ergast is discontinued at the end of 2024.

**Comment by serrq** [comment]
Okay, thanks!
